--- gcc_gen.sh.orig	2008-11-04 00:10:23.000000000 +0100
+++ gcc_gen.sh	2010-04-27 19:39:23.000000000 +0200
@@ -81,8 +81,8 @@
 		echo "	\$(LINKER) -o $subdir/lib$libname.so \$(LDFLAGS)  \$(ICU_LDFLAGS) $all_obj \$(ICU_LIBS) \$(LIBS)" >> $tout
 	else
 		echo ./$subdir/lib$libname.a : $all_obj >> $tout
-		echo "	ar -r $subdir/lib$libname.a $all_obj" >> $tout
-		echo "	-ar -s $subdir/lib$libname.a" >> $tout
+		echo "	/usr/i586-mingw32msvc/bin/ar -r $subdir/lib$libname.a $all_obj" >> $tout
+		echo "	-/usr/i586-mingw32msvc/bin/ar -s $subdir/lib$libname.a" >> $tout
 	fi
 	echo "" >> $tout
 }
@@ -131,8 +131,10 @@
 # LIBS=     additional library files
 
 # compiler:
-CXX=g++
-LINKER=g++ -shared
+GCC_ROOT = /usr/i586-mingw32msvc
+CXX=i586-mingw32msvc-gcc
+LINKER=i586-mingw32msvc-gcc
+INCLUDES=-I/usr/i586-mingw32msvc/include
 
 #
 # compiler options for release build:
